### ISSUE TITLE
About us

### ISSUE THREAD

**Issue by victorvalle** [liferay-design/liferay.design#546]
It would be nice to have an About Us section or a place where we describe the work we do. Or how to we see product design at Liferay. The kind of projects we participate in...

What do you think?

**Comment by plhnk** [comment]
![giphy (1)](https://user-images.githubusercontent.com/10226086/65158040-8c3ca180-d9e6-11e9-9199-557eac46a2cb.gif)

**Comment by victorvalle** [comment]
- How we are structured: comm, prod
- How we integrate in the product: multidisciplinary teams
- Lexicon
- How do we use and interact with Lexicon
- Our culture, open space to write and post about design...

Something short, explanatory and catchy.

**Comment by ryconn888** [comment]
I agree we need this page -- but i don't think we can combine this with the careers/growth page we discussed. These need to be two separate pages IMO. I'll do mockups for both just in case we want two distinct pages.
